src/Vision/ImageAnnotations.php

```diff
@@ -8,7 +8,11 @@ class ImageAnnotations
 {
     private const IDENTITY_DOCUMENT_LABEL = 'Identity document';
     private const NATIONAL_IDENTITY_CARD_LABEL = 'National identity card';
-    private const FRENCH_IDENTITY_CARD_LABEL = 'carte d identité française';
+    private const FRENCH_LANGUAGE = 'French language';
+    private const FRENCH_IDENTITY_CARD_LABELS = [
+        'carte d identité française',
+        'carte d identité verso',
```

Comment:
on my test I had this, so I added it here because I think it is valid too. Do you agree ?

Reply:
i am not so sure.
we specifically ask for the first side of the card, not the other side.

Reply:
reverted